test/unit/org/apache/cassandra/db/filter/ColumnFilterTest.java

```diff
@@ -208,6 +241,38 @@ private static void testRoundTrip(ColumnFilter columnFilter, ColumnFilter expect
         Assert.assertEquals(deserialized, expected);
     }
 
+    @Test
+    public void testToString()
+    {
+        TableMetadata metadata = TableMetadata.builder("ks", "table")
```

Comment:
As [suggested in the PR for 3.0](https://github.com/apache/cassandra/pull/890#discussion_r578410628), we could include a multi-cell column to the table metadata to test how subselections are printed.